conftest.py

```diff
@@ -0,0 +1,9 @@
+def pytest_addoption(parser):
+    parser.addoption('--including-search', action='store_true', dest="searchtests",
```

Comment:
We should document this in the testing docs: https://github.com/rtfd/readthedocs.org/blob/master/docs/tests.rst so that folks know about it.

Reply:
Yeah. documenting